src/modules/job-manager/urgency.c

```diff
@@ -120,7 +123,8 @@ void urgency_handle_request (flux_t *h,
             goto error;
         alloc_queue_reorder (ctx->alloc, job);
         if (alloc_queue_recalc_pending (ctx->alloc) < 0)
-            flux_log_error (h, "%s: alloc_queue_recalc_pending", __FUNCTION__);
+            flux_log_error (h, "%s: alloc_queue_recalc_pending",
+                            __FUNCTION__);
```

Comment:
There will already be an error logged (two actually) if `alloc_queue_recalc_pending()` fails, and I thnk the only way it can fail is if a fire and forget RPC fails (like from ENOMEM).  Let's skip logging an error here too.